tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,98 +1,2553 @@
+    @if $very-very-very-very-very-very-very-very-very-very-very-long-var == 0 and $very-very-very-very-very-very-very-very-very-very-very-long-var == 0 {}
+    @if $base-font-size != 16px or $base-line-height != 24px or $base-unit != 'em' or $h1-font-size != 2 * $base-font-size or $h2-font-size != 1.5 * $base-font-size or $h3-font-size != 1.17 * $base-font-size or $h4-font-size != 1 * $base-font-size or $h5-font-size != 0.83 * $base-font-size or $h6-font-size != 0.67 * $base-font-size or $indent-amount != 40px {}
+    @if (str-slice($item, 0, 1) == ":") {}
+    @if (str-slice($item, 0, 3) == " : ") {}
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@if $media == phonePortrait {
+  $k: 0.15625;
+} @else if $media == phoneLandscape {
+  $k: 0.08803;
+} @else if $media == tabletPortrait {
+  $k: 0.065106;
+}
+p {
+  @if $type == ocean {
+  }
+  @if $type==ocean {
+  }
+  @if $type == ocean {
+  }
+  @if $type == ocean {
+  }
+  @if $type == ocean {
+  }
+  @if $type == ocean {
+  }
+  @if (($type) == (ocean)) {
+  }
+  @if (($type)==(ocean)) {
+  }
+  @if (($type) == (ocean)) {
+  }
+  @if (($type) == (ocean)) {
+  }
+  @if (($type) == (ocean)) {
+  }
+  @if (($type) == (ocean)) {
+  }
+  @if $type == ocean {
+    color: blue;
+  } @else if $type == matador {
+    color: red;
+  } @else if $type == monster {
+    color: green;
+  } @else {
+    color: black;
+  }
+  @if $type == ocean {
+  } @else if $type == matador {
+  }
+  @if $type==ocean {
+  } @else if $type==matador {
+  }
+  @if $type == ocean {
+  } @else if $type == matador {
+  }
+  @if $type == ocean {
+  } @else if $type == matador {
+  }
+  @if $type == ocean {
+  } @else if $type == matador {
+  }
+  @if $type == ocean {
+  } @else if $type == matador {
+  }
+  @if $type == ocean {
+  } @else if $type == matador {
+  }
+  @if $very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-variables ==
+    $very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-variables
+  {
+    color: blue;
+  } @else if
+    $very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-variables ==
+    $very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-variables
+  {
+    color: red;
+  } @else if
+    $very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-variables ==
+    $very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-variables
+  {
+    color: green;
+  } @else {
+    color: black;
+  }
+  @if $very-very-very-long-var == 0 and $very-very-very-long-var == 0 {
+  }
+  @if $very-very-very-very-very-very-long-var ==
+    0 and
+    $very-very-very-long-var ==
+    0
+  {
+  }
+  @if $very-very-very-very-very-very-very-very-very-very-very-long-var ==
+    0 and
+    $very-very-very-very-very-very-very-very-very-very-very-long-var ==
+    0
+  {
+  }
+  @if $base-font-size !=
+    16px or
+    $base-line-height !=
+    24px or
```

Comment:
Same thing here: Would be great if we could keep the comparisons together.

Reply:
@lydell :+1: